internal/app/machined/pkg/runtime/v1alpha1/board/rpi_4/rpi_4.go

```diff
@@ -14,11 +15,16 @@ import (
 	"github.com/talos-systems/talos/pkg/machinery/constants"
 )
 
-var configTxt = []byte(`arm_64bit=1
+// https://www.raspberrypi.org/documentation/configuration/config-txt/
+var configTxt = []byte(strings.TrimSpace(`
+
+arm_64bit=1
 enable_uart=1
 kernel=u-boot.bin
 dtoverlay=disable-bt
-`)
+hdmi_safe=1
+
+` + "\n"))
```

Comment:
That's a trick I use to separate inline data like YAML fragments from the code. Not sure if you like it or not.

Reply:
👍 ... I believe 1.16 is going to have a way to do something like this.